docs/tutorials/plugin-provider.md

```diff
@@ -12,6 +12,19 @@ The idea behind it is that providers can be implemented in a separate program: t
 
 Providers implementing the HTTP API have to keep in sync with changes to the Go types `plan.Changes` and `endpoint.Endpoint`. We do not expect to make significant changes to those types given the maturity of the project, but we can't exclude that changes will need to happen. We commit to publishing changes to those in the release notes, to ensure that providers implementing the API can keep providers up to date quickly.
 
+## Implementation requirements
+
+The following table represents the methods to implement mapped to their HTTP method and route.
+
+| Provider method | HTTP Method | Route |
+| --- | --- | --- |
+| Records | GET | /records |
+| ApplyChanges | POST | /records |
+| PropertyValuesEqual | GET | /propertiesvaluesequal |
```

Comment:
```suggestion
| PropertiesValuesEqual | GET | /propertiesvaluesequal |
```

Reply:
Rejecting as the name of the interface is `PropertyValuesEqual`.

Reply:
Ah understood! Thanks!